question-service/app/crud/questions.py

```diff
@@ -39,14 +39,22 @@ async def delete_question(question_id: str):
 
 async def update_question_by_id(question_id: str, question_data: UpdateQuestionModel):
```

Comment:
Would be difficult to identify what this function returns. Currently, it can return `None`, `str` and `QuestionModel` which may make it difficult to understand the code

Reply:
Yep, we need some form error handling for this and other end points too. Instead of returning `None`/`str`, should throw error instead